lib/ui/view/channel/channel_members/channel_members_list.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/view/plugins/widgets/custom_plugin_list_tile.dart';
+import 'package:stacked/stacked.dart';
+import 'package:velocity_x/velocity_x.dart';
+import '../shared_widgets.dart';
+import 'channel_members_list_model.dart';
+
+class ChannelMembersList extends StatelessWidget {
+  const ChannelMembersList({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<ChannelMembersModel>.reactive(
+        viewModelBuilder: () => ChannelMembersModel(),
+        builder: (context, viewModel, child) {
+          return Scaffold(
+            appBar: AppBar(
+              title: '#teamsocrates'
+                  .text
+                  .textStyle(GoogleFonts.lato())
+                  .color(Color(0xFF242424))
+                  .bold
+                  .size(20.0)
+                  .make(),
+              backgroundColor: Colors.white,
+              elevation: 1.5,
+              leading: IconButton(
+                icon: Icon(
+                  Icons.clear,
+                  color: Color(0xFF333333),
+                  // size: 24,
+                ),
+                padding: EdgeInsets.only(left: 18),
+                onPressed: () {},
+                iconSize: 25.0,
+              ),
+              actions: [
+                InkWell(
+                  child: 'Edit'
```

Comment:
I thought there is a general style to be use, this won't scale when the time comes. Please use the style guide and remove any dependency on velocityX.